core/core.go

```diff
@@ -117,6 +131,21 @@ func (e *Emulator) atBreakpoint() bool {
 	return e.breakpoint != 0 && uint16(e.breakpoint&0xFFFF) == e.CPU.PC && uint8(e.breakpoint>>16) == e.CPU.K
 }
 
+func (e *Emulator) SetRegisterBreakpoint(register string) {
+	e.registerBreakpoint = register
+}
+
+func (e *Emulator) atRegisterBreakpoint(register string) bool {
+	return e.registerBreakpoint != "" && register == e.registerBreakpoint
+}
+
+func (e *Emulator) maybePause(register string) {
```

Comment:
nit: can we rename this into `handleRegisterBreakpoint` or maybe something else ? (the name does not really tell what the function is doing)

Reply:
Yes right